src/main/resources/beam-template.conf

```diff
@@ -119,7 +119,7 @@ beam.agentsim.agents.rideHailTransit.modesToConsider = "MASS"
 # Ride Hailing Params
 beam.agentsim.agents.rideHail.initialization.initType = "PROCEDURAL" # Other possible values - FILE
 beam.agentsim.agents.rideHail.initialization.filePath = ${beam.inputDirectory}"/ride-hail-fleet.csv"
-beam.agentsim.agents.rideHail.initialization.parking.filePath = ${beam.inputDirectory}"/ride-hail-parking.csv"
+beam.agentsim.agents.rideHail.initialization.parking.filePath = ""
```

Comment:
why are we cleaning only this default path and not all default paths?

Reply:
We are overriding other default path in user defined config. So its not mandatory to remove default files for them